index.html

```diff
@@ -1,175 +0,0 @@
-<!DOCTYPE html>
-<html lang="pt-br">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="estilos/reset.css">
-    <link rel="stylesheet" href="estilos/style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Righteous&family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet">
-</head>
-<body>
-    <!-- Cabeçalho do Site -->
-    <header>
-        <div class="logo">
-            <h1>DrivenEats</h1>
-            <p>Sua comida em 6 minutos</p>
-        </div> 
-    </header>
-
-    <!-- Conteúdo Principal -->
-    <main>
-
-        <!-- Itens - Pratos -->
-        <section class="itens">
-            <h2>Primeiro, seu prato</h2>
-            <div class="escolhas pratos" data-identifier="dishes">
-
-                <div class="item prato1" onclick="selecionarPrato('prato1','Fragon Yin Yang', 14.90)" data-identifier="food-option">
-                    <img src="imagens/frango_yin_yang 1.png" alt="Frango Yin Yang">
-                    <h3 data-identifier="food-title">Frango Yin Yang</h3>
-                    <p>Um pouco de batata, um pouco de salada</p>
-                    <span data-identifier="food-price">R$ 14,90</span>
-                    <ion-icon name="checkmark-circle-outline"></ion-icon>
-                </div>
-
-                <div class="item prato2" onclick="selecionarPrato('prato2','Frango Yin Yang', 12.90)" data-identifier="food-option">
-                    <img src="imagens/frango_yin_yang 1.png" alt="Frango Yin Yang">
-                    <h3 data-identifier="food-title">Frango Yin Yang</h3>
-                    <p>Um pouco de batata, um pouco de salada</p>
-                    <span data-identifier="food-price">R$ 12,90</span>
-                    <ion-icon name="checkmark-circle-outline"></ion-icon>
-                </div> 
-
-                <div class="item prato3" onclick="selecionarPrato('prato3', 'Frango Yin Yang', 15.90)" data-identifier="food-option">
```

Comment:
usando só o this já importaria tudo, não precisaria descrever como você fez nos parênteses